connectors/cassandra/datastore_multi.go

```diff
@@ -0,0 +1,165 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
This was copied from the closed source connector. At some point we will be adding multi support, but yeah, we should just delete this for now.